### PR TITLE
thesilence - doc updates

### PR DESCRIPTION
**Pull request by therealsilence**
- add docs for new storm commands
- add glossary
- minor tweaks

### CODE REVIEW

File: docs/synapse/glossary.rst [hunk 1]
```diff
@@ -0,0 +1,717 @@
...
+A graph is a mathematical structure used to model pairwise relations between objects. Graphs consist of vertices (or nodes) that represent objects and edges that connect exactly two vertices in some type of relationship. Nodes and edges in a graph are typically represented by dots or circles conneted by lines.
+
+See :ref:`bkd-graphs-hypergraphs` for additional detail on graphs and hypergraphs.
+
+.. _gloss-graph-directed:
+
+Graph, Directed
+---------------
+
+A directed graph is a :ref:`gloss-graph` where the edges representing relationships between nodes have a "direction". Given node X and node Y connected by edge E, the relationship is valid for X -> E -> Y butnot Y -> E -> X. For example, the relationship "Fred owns bank account #01234567" is valid, but "bank account #01234567 owns Fred" is not. Nodes and edges in a directed graph are typically represented by dots or circles connected by arrows.
```
Comment (Cisphyx) [inline docs/synapse/glossary.rst]: "X -> E -> Y butnot Y -> E -> X"

File: docs/synapse/glossary.rst [hunk 2]
```diff
@@ -0,0 +1,717 @@
...
+-------
+
+TBD
+
+.. _gloss-pivot:
+
+Pivot
+-----
+
+Within Synapse, one of the three primary methods for interacting with data in a :ref:`gloss-cortex`. A pivot operation allows navigation of the hypergraph following a lift operation. A pivot moves from a set of nodes with one or more properties with specified value(s) to a set of nodes with a property having the the same value(s).  Compare with :ref:`gloss-lift` and :ref:`gloss-filter`.
```
Comment (Cisphyx) [inline docs/synapse/glossary.rst]: Double "the" in "property having the the same value(s)"

File: docs/synapse/glossary.rst [hunk 3]
```diff
@@ -0,0 +1,717 @@
...
+
+.. _gloss-prop-secondary:
+
+Property, Secondary
+-------------------
+
+Within Synapse, secondary properties are optional properties that provide additional detail about a :ref:`gloss-form`. Within the data model, secondary properties may be defined with optional constraints, such as:
+
+  - Whether the property is read-only once set.
+  - Whether a default value should be set for the property if no value is specified.
```
Comment (Cisphyx) [inline docs/synapse/glossary.rst]: Sounds like we might be getting rid of default values for 0.2.0?

File: docs/synapse/userguides/storm_ref_cmd.ipynb [hunk 4]
```diff
@@ -437,9 +472,101 @@
     "\n",
     "- ``delnode`` operates on the output of a previous Storm query.\n",
     "- ``delnode`` will attempt to perform some basic sanity-checking to help prevent egregious mistakes. For example, ``delnode`` will return an error if you attempt to delete a node that is still referenced by another node (such as an ``inet:fqdn`` that is referenced by an ``inet:dns:a`` node). Similarly, delnode will return an error if you attempt to delete a ``syn:tag`` node if that tag is still applied to other nodes. **However, delnode cannot prevent all mistakes.**\n",
+    "  \n",
+    "  - Edge / digraph nodes (such as ``edge:refs`` or ``edge:has``) are **exempt** from these \"reference checks\", It is possible to delete an edge node where ``n1`` and / or ``n2`` still exist and without affecting ``n1`` or ``n2``. This is generally desired behavior as the relationship between the nodes (the edge) may need to be removed without affecting the nodes themselves.\n",
```
Comment (Cisphyx) [inline docs/synapse/userguides/storm_ref_cmd.ipynb]: I think the comma at "from these \"reference checks\", " should be a period, assuming I'm reading it correctly.

**Comment by therealsilence** [comment]
Addressed comments / typos.

**Pull request merged**